.meta/transforms/transaction.toml

```diff
@@ -0,0 +1,248 @@
+[transforms.transaction]
+title = "Transaction"
+allow_you_to_description = "combines events of a matching transaction into a single event"
+beta = true
+common = false
+function_category = "aggregate"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/fields/_component_options.toml", type: "transform", name: "transaction") %>
+
+[transforms.transaction.options.expire_after_ms]
+type = "int"
+common = false
+required = false
+default = 30000
+description = """\
+A maximum period of time to wait before a transaction should be considered \
+complete after not having received another event.\
+"""
+
+[transforms.transaction.options.flush_period_ms]
+type = "int"
+common = false
+required = false
+default = 1000
+description = """\
+Controls the frequency that Vector checks for (and flushes) expired \
+transactions.\
+"""
+
+[transforms.transaction.options.identifier_fields]
+type = "[string]"
+common = true
+required = false
+default = []
+examples = [["request_id"],["user_id","transaction_id"]]
+description = """\
+An ordered list of fields to distinguish transactions by. Each transaction has \
+a separate event combining state, and these fields should be used to prevent
+events from unrelated transactions from combining.\
+"""
+
+[transforms.transaction.options.merge_strategies]
+type = "table"
+common = false
+required = false
+description = """\
+A map of field names to custom merge strategies. For each field specified this \
+strategy will be used for combining events rather than the default behavior.
+
+The default behavior is as follows:
+
+1. The first value of a string field is kept, subsequent values are discarded.
+2. For timestamp fields the first is kept and a new field `[field-name]_end` is
+   added with the last received timestamp value.
+3. Numeric values are added.\
+"""
+
+[transforms.transaction.options.merge_strategies.children."`[field-name]`"]
+type = "string"
+required = true
+common = true
+examples = [
+  {method="discard"},
+  {path="discard"}, 
+  {duration_ms="sum"},
+  {query="array"}
+]
+description = "The custom merge strategy to use for a field."
+
+[transforms.transaction.options.merge_strategies.children."`[field-name]`".enum]
+"array" = "Each value is appended to an array."
+"concat" = "Concatenate each string value (delimited with a space)."
+"discard" = "Discard all but the first value found."
+"sum" = "Sum all number values."
+"max" = "The maximum of all number values."
+"min" = "The minimum of all number values."
+
+[transforms.transaction.options.ends_when]
```

Comment:
Are these the same as the swimlanes conditions? Can we mention that on both if so? (To make it clear folks can cross their knowledge here)